tests/utils/test_logger.py

```diff
@@ -28,23 +29,34 @@ def setUpClass(cls) -> None:
         configuration.freeze()
         cls.config = configuration.get_config()
         registry.register("config", cls.config)
-        cls.writer = Logger(cls.config)
+        setup_output_folder.cache_clear()
+        setup_logger.cache_clear()
+        cls.writer = setup_logger()
 
     @classmethod
     def tearDownClass(cls) -> None:
         # Cleanup temp working dir.
-        for handler in cls.writer._logger.handlers:
+        for handler in cls.writer.handlers:
             handler.close()
         if cls._tmpdir is not None:
             # set ignore_errors as Windows throw error due to Permission
             shutil.rmtree(cls._tmpdir, ignore_errors=True)
```

Comment:
Not sure what you mean by `long failing windows logger tests` in your PR description. Those were already passing earlier. One issue was regarding permissions removing the logger folder. If that is fixed by these changes, then we can remove `ignore_errors=True` here.

Reply:
The log files with ":" were some how still failing with OSError. I fixed them by changing ":" to "_".

Reply:
Thanks. PR #352 did fix some log file names. Maybe it missed some.